protool/__init__.py

```diff
@@ -65,6 +65,24 @@ def profile_type(self) -> ProvisioningType:
 
         raise Exception("Unable to determine provisioning profile type")
 
+    @property
```

Comment:
This does a lot for a property. Keeping it as a method/function makes it clear that it will do some amount of work.